### ISSUE TITLE
pm2 web command not found

### ISSUE THREAD

**Issue by aalok05** [Unitech/pm2#4805]
## What's going wrong?
Can't find pm2 web interface

## How could we reproduce this issue?
Run `pm2 web` command.
Response:  `pm2 web command not found`
pm2 version : 4.4.0

What's the alternative for `pm2 web` ? Is there a way to get metrics in JSON format?

**Comment by mikebevz** [comment]
any news on that?

**Comment by aalok05** [comment]
> 
> 
> any news on that?

Nope. I think they want people to use the SaaS service, which is why this functionality has been removed.
